packages/integration-tests/src/helper.ts

```diff
@@ -61,8 +61,12 @@ export class Helper {
     return new Promise((resolve) => setTimeout(resolve, ms));
   }
 
-  async performGETrequest(apiRoute: string) {
-    return request(environment.blockExplorerAPI).get(apiRoute);
+  async performGETrequest(apiRoute: string, network = "local") {
```

Comment:
This update of the code is designed to be able to call Sepolia API.
Why?
Because for Contracts API we need to have contracts deployed and **verified**.
Why Sepolia?
Because we don't have verification flow implemented for local environments. As a result, we can't implement it. I tried and appeared that it is not supported at all for now.
So?
I decided to use Sepolia contract, that was deployed by me, and won't be changed.
Why such implementation?
Because by default I left network = local, so that we don't need to update all other usages of this function.

Reply:
@abilevych do you see this comment?